app/models/katello/system.rb

```diff
@@ -69,16 +62,18 @@ class System < Katello::Model
     before_update :update_foreman_host, :if => proc { |r| r.environment_id_changed? || r.content_view_id_changed? }
 
     scope :in_environment, ->(env) { where('environment_id = ?', env) unless env.nil? }
-    scope :completer_scope, ->(options) { readable(options[:organization_id]) }
```

Comment:
So this is actually used I think on the Role filters page.  Without this I get an error when trying to autocomplete for content hosts on that page ("name = "):

```
PG::UndefinedColumn: ERROR:  column katello_systems.organization_id does not exist
LINE 1: ...llo_systems"."name" FROM "katello_systems"  WHERE "katello_s...
```
granted with this I get an error because readable doesn't take an argument as I said.  So there is something going on here that i think needs to be resolved.